Gemfile.lock

```diff
@@ -240,6 +244,7 @@ GEM
 
 PLATFORMS
   ruby
+  x64-mingw32
```

Comment:
Just out of curiosity what is `x64-mingw32`?

Reply:
🤔 I don't really know what that is, maybe it was added because I did it on windows.
I don't know but if you want I can remove it.